pages/exhibition/book-your-stand.jsx

```diff
@@ -14,7 +14,7 @@ const generalPage = (props) => {
           mb: 6,
         }}
       >
-        <div>book your stand</div>
+        <div>This is done by tarek</div>
```

Comment:
can you change this to use the <SectinoTitle> component 
return it to the original text but use the component to make it look like a section title

Reply:
okay , I will do it right now